_data/payments.yml

```diff
@@ -105,6 +105,12 @@ websites:
       software: Yes
       doc: https://privacy.com/faq
 
+    - name: Ravelin
+      url: https://www.ravelin.com/
+      img: Ravelin.png
```

Comment:
Could you please rename the image to be all lower-case, and change this line to reflect that?